plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -408,7 +415,13 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         }
     }
 
-    private void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata)
+    @Override
+    public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorTableLayout> layout, RetryMode retryMode)
+    {
+        return createTable(session, tableMetadata);
```

Comment:
Does this also need a defensive check to verify query retries are not enabled?

Reply:
I think so. Thanks for catching that. Do you want to send the PR?

Reply:
https://github.com/trinodb/trino/pull/13812